website/frontend/templates/base.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{{ _('Official website for MusicBrainz Picard, a cross-platform music tagger written in Python.') }}">
     <meta name="author" href="{{ url_for('humans.show_humans') }}">
-    <link rel="icon" type="image/png" href="{{ url_for('static', filename='img/picard-icon.png') }}">
```

Comment:
This is actually one case where it's better to use PNG. See http://caniuse.com/#feat=link-icon-svg.